packages/photon/package.json

```diff
@@ -40,8 +46,8 @@
 		"@automattic/calypso-typescript-config": "workspace:^"
```

Comment:
Missing dependency on `typescript`?

Reply:
yep! Relatedly, looks like we missed it for the `viewport` migration as well.

Reply:
Should `@automattic/calypso-typescript-config` have a peer dependency on typescript?

Reply:
`@automattic/calypso-typescript-config` exports a JS-only config as well. The name is misleading :(